.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: pushd ../frontend/ && yarn install && popd
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Java Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-build-${{ hashFiles('*.gradle', 'gradle/dependency-locks/*') }}
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/java/linux64 --post-data "token=$TOKEN&project=CDCgov/prime-data-input-client" -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+        env:
+          TOKEN: ${{ secrets.COVERITY_TOKEN }}
```

Comment:
Can just inline this into the `run` command.

Reply:
Seems like it will be more consistent to inject everything as an environment variable, otherwise,  But we can change if that makes you more comfortable.

Reply:
It's not a big deal one way or the other, but anything that adds a level of indirection adds an opportunity for somebody to misunderstand it or mess it up on edit. More generally, I think it's a better practice to avoid creating env variables unless we specifically want the information in the environment, but that's possibly just me being weird.

Reply:
I think it's fine as long as it's consistent. I went ahead and inlined the variables, since I had inlined the git sha command.